Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandMapModel.swift

```diff
@@ -257,7 +257,7 @@ class OnDemandMapModel: ObservableObject, Identifiable {
     }
     
     /// Updates the status based on the download result of the mobile map package.
-    private func updateDownloadStatus(for downloadResult: Result<GenerateOfflineMapResult, any Error>) {
+    private func updateDownloadStatus(for downloadResult: Result<GenerateOfflineMapResult, Error>) {
```

Comment:
Consider leaving `any` in place as it will be [required](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0335-existential-any.md#proposed-solution) in a future version of Swift. Same elsewhere.

Reply:
Good point. They don't give "this will be an error in a future Swift language mode" warnings for now but it makes sense to use `any`.